app/src/main/java/com/example/parkingsystem/utilities/DateUtil.java

```diff
@@ -0,0 +1,13 @@
+package com.example.parkingsystem.utilities;
+
+import java.util.Date;
+
+public class DateUtil {
```

Comment:
is this class used? it's good to have this kind of date validator, but there is no need to do it rigth now

Reply:
The class was not being used, hence it has been removed.